webapp/src/components/backstage/template_selector.tsx

```diff
@@ -25,12 +26,35 @@ export interface PresetTemplate {
     template: DraftPlaybookWithChecklist;
 }
 
-export const PresetTemplates: PresetTemplate[] = [
+const preprocessTemplates = (presetTemplates: PresetTemplate[]): PresetTemplate[] => (
+    presetTemplates.map((pt) => ({
+        ...pt,
+        template: {
+            ...pt.template,
+            num_stages: pt.template?.checklists.length,
+            num_actions:
+				1 + // Channel creation is hard-coded
```

Comment:
Odd that editorconfig didn't pick up on that! (tabs-spaces)
```suggestion
                1 + // Channel creation is hard-coded
```